base/shell/explorer/appbar.cpp

```diff
@@ -198,22 +200,33 @@ static UINT_PTR handle_appbarmessage(DWORD msg, _AppBarData *abd)
         }
         return TRUE;
     case ABM_GETSTATE:
-        FIXME("SHAppBarMessage(ABM_GETSTATE): stub\n");
-        return ABS_ALWAYSONTOP | ABS_AUTOHIDE;
+        TRACE("SHAppBarMessage(ABM_GETSTATE)\n");
```

Comment:
Why do these mention SHAppBarMessage?

Reply:
Because `SHAppBarMessage` is the documented API for all of this. It lives in shell32 and sends a message here for processing.